docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc

```diff
@@ -46,19 +46,17 @@ instead of the primary headers.
 This API creates a {dfanalytics-job} that performs an analysis on the source 
 index and stores the outcome in a destination index.
 
-The destination index will be automatically created if it does not exist. The 
+If it does not exist, the destination index is created automatically. The 
 `index.number_of_shards` and `index.number_of_replicas` settings of the source 
-index will be copied over the destination index. When the source index matches 
-multiple indices, these settings will be set to the maximum values found in the 
-source indices.
+index are copied over the destination index. When the source index matches 
+multiple indices, these settings are set to the maximum values found in the 
+source indices. The mappings of the source indices are also copied over to the
+destination index. If the mappings of any of the fields don't match the source
+indices, however, the attempt fails with an error message.
+//TBD Does this cause the job creation to fail? What are the implications?
```

Comment:
We copy the mappings to the dest index when the `_start` API is called. Indeed, if the mappings are conflicted, the start API fails with an error. But during job creation we don't check.